cmd/tsbs_generate_queries/databases/timescaledb/devops.go

```diff
@@ -70,19 +73,32 @@ const goTimeFmt = "2006-01-02 15:04:05.999999 -0700"
 //
 // SELECT minute, max(metric1), ..., max(metricN)
 // FROM cpu
-// WHERE (hostname = '$HOSTNAME_1' OR ... OR hostname = '$HOSTNAME_N')
-// AND time >= '$HOUR_START' AND time < '$HOUR_END'
-// GROUP BY minute ORDER BY minute ASC
+// WHERE
+//      (hostname = '$HOSTNAME_1' OR ... OR hostname = '$HOSTNAME_N')
+// 	AND time >= '$HOUR_START'
+// 	AND time < '$HOUR_END'
+// GROUP BY minute
+// ORDER BY minute ASC
 func (d *Devops) GroupByTime(qi query.Query, nHosts, numMetrics int, timeRange time.Duration) {
 	interval := d.Interval.RandWindow(timeRange)
 	metrics := devops.GetCPUMetricsSlice(numMetrics)
 	selectClauses := d.getSelectClausesAggMetrics("max", metrics)
 
-	sql := fmt.Sprintf(`SELECT time_bucket('1 minute', time) AS minute,
-    %s
-    FROM cpu
-    WHERE %s AND time >= '%s' AND time < '%s'
-    GROUP BY minute ORDER BY minute ASC`,
+	sql := fmt.Sprintf(`
```

Comment:
I would prefer these formats _not_ to change since I find this new way much more difficult to read.

Reply:
Ok, agreed. In general, I've rolled back all Timescale changes because it is a rudiment of of code reverse and understanding.
Summary: updated

Reply:
I don't see these changes yet?

Reply:
Just pushed, I am writing this notes along with code review